frontend/server/package-lock.json

```diff
@@ -1769,7 +1769,8 @@
         },
         "ansi-regex": {
           "version": "2.1.1",
-          "bundled": true
+          "bundled": true,
```

Comment:
I don't think you installed any new dependencies. So it's better to revert changes here.